src/Phpbb/TranslationValidator/Validator/FileValidator.php

```diff
@@ -591,8 +591,8 @@ public function validateNoPhpClosingTag($originFile)
 		$fileContents = str_replace("\r\n", "\n", $fileContents);
 		$fileContents = str_replace("\r", "\n", $fileContents);
 
-		// Does the file contain anything after the last ");"
-		if (substr($fileContents, -3) !== ");\n")
+		// Does the file contain anything after the last ");" or "];" (for short array syntax)
+		if (substr($fileContents, -3) !== ");\n" && substr($fileContents, -3) !== "];\n")
```

Comment:
Please check the version for the new part, that is only allowed with `$this->phpbbVersion === '3.2'`